02_retention_curves/saved_views/cohort_table.yml

```diff
@@ -1,41 +1,38 @@
-glean: '1.0'
+glean: "1.0"
```

Comment:
<img width="534" alt="Screenshot 2023-05-02 at 6 27 12 PM" src="https://user-images.githubusercontent.com/6391516/235799007-076e4683-a17e-4dbd-aebe-9aca82e64a14.png">

I see this error in the preview, expected?